src/js/tabs/firmware_flasher.js

```diff
@@ -220,8 +220,26 @@ firmware_flasher.initialize = function (callback) {
                 return;
             }
 
+            data.osdProtocols = [
```

Comment:
```suggestion
            data.osdProtocols = data.generalOptions.filter(option => option.group === 'OSD');
```

Reply:
You can map to a new item, using the groupedName if you want to change the name to a different name.

Reply:
https://build.betaflight.com/api/options

Reply:
you will also need to cater for the "none" option... so maybe a helper function to add this, and map. note the none option should have an empty value.